src/components/Player/Pages/Overview/Overview.jsx

```diff
@@ -262,6 +279,50 @@ const countValidRecentMatches = matches => matches.filter(match => match.game_mo
 const getValidRecentMatches = matches => matches.filter(match => match.game_mode !== 19)
   .slice(0, MAX_MATCHES_ROWS);
 
+const filterCounts = (counts) => {
+  const countMap = {
+    is_radiant: [],
+    game_mode: [],
+    patch: [],
+    region: [],
+  };
+
+  const limitCount = (key, field, lim) =>
+    counts[key].list.sort((a, b) => (
+      a[field] <= b[field] ? 1 : -1
+    )).slice(0, lim);
+
+  Object.keys(counts).forEach((key) => {
+    switch (key) {
+      case 'is_radiant':
```

Comment:
maybe we should show the top two for each, and put each in its own column?  not sure if that would make it too tall.

Reply:
I think it would be too tall.

Reply:
Think we could still do top two for each but leave it one row.  That would be 8 gauges (4x2)

Reply:
could possibly also fit one more set of gauges, maybe the top two lane_role counts (skipping unknown)